script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -886,29 +917,34 @@ I let out a sigh and reply.
 [sha:6283d10765f1fbc8f997eab7e56f9e844df877fa]{
 -- Page 561, Offset 16155.
 -- 無邪気に褒められてもこっちは素直に喜べない。
-Even her innocent praise is not enough for me to feel entirely happy.
+Even though her praise is honest,
+ I doesn't cheer me up at all.
 }
 [sha:61571e41f1baf0bf836207cc0267b55b5d687cf9]{
 -- Page 561, Offset 16156.
 -- 殺人の才能が認められるのは、おまえはろくでなしだ、と言われているようなものだからだ。
-Because having my murderous talents acknowledged is basically like being told that I'm an awful person.
+Being complimented on my murderous talents feels the same as being told that I'm an awful person.
 }
 [sha:942ebb596bfb992ec820d2d38679cc204c9735af]{
 -- Page 562, Offset 16157.
 -- 「でも、その眼と同じぐらい貴方自身もおかしな人ね。
-"Still, you yourself is almost just as much of a mystery as those eyes are.
+"Still, you yourself are just as much a mystery as those eyes are.
 }
 [sha:60cb2f6b43e56cb01230ce3a1f049b1f91c70ad9]{
 -- Page 562, Offset 16158.
 -- 直前まで逃げ腰だったのに、ナイフを握って、殺すと決めた途端、迷い無く死者を解体するんだもの。
-You were basically on the verge of running away right until you took your knife.
- Once you decided to kill, you dismembered those Dead without hesitation.
+You were ready to turn tail and run right up until the moment you took out your knife.
+ But once you decided to kill, you dismembered those Dead without a second thought.
 }
 [sha:9b8316937dc6a08e0f43f89e735ddeca6ca9888e]{
 -- Page 562, Offset 16159.
 -- そういうところ、わたしと似ているのかも。なのに人間よりも人間らしいなんて、すごく不思議」
-In that respect, you almost resemble me.
- Yet even so, it's super strange how you're almost more human-like than humans are."
+// 人間よりも人間らしい
+// More human than humans (class of his own) or more human than most humans (he's just a great guy)?
+// Leaning first since it's more tokubetsu
+You almost resemble me in that regard.
+ Yet despite that, you're more human than humans are.
```

Comment:
I lean to the latter, though I think rather than him being a 'great guy', it's more about having strongly human convictions about life, death and killing.